OOP_HW1/src/main/java/Order.java

```diff
@@ -0,0 +1,28 @@
+public class Order {
+    private TicketType type;
+    private int quantity;
+
+    public Order() {
+    }
+
+    public Order(TicketType t, int q) {
+        type = t;
+        quantity = q;
+    }
+
+    public void setTicketType(TicketType t) {
+        type = t;
+    }
+
+    public void setQuantity(int q) {
+        quantity = q;
+    }
```

Comment:
You don't need it. The whole homework only need the constructor to construct object. The order won't be changed when you execute any function in `TicketSystem`.